cmd/state/forward.go

```diff
@@ -128,7 +129,7 @@ func ensureForwardExists(binary string, versionInfo *projectfile.VersionInfo, ou
 		DesiredVersion: desiredVersion,
 	}
 
-	info, err := up.Info()
+	info, err := up.Info(context.Background())
```

Comment:
Not for this story, but... We have a greater issue; We need to wrangle our dependency on an HTTP client. In order to get the benefit of connection reuse, as well as putting a backstop timeout at the transport level, we should be passing around a common client that can have more aggressive per-request timeouts applied via context.Context as you've done here. Anyway, this is all to say that it feels wrong to me that we are making any HTTP calls without some timeout given. Again, for another story.

Reply:
It would be kind of nice if we had more timeouts.  On the other hand, it is (still) a console application, so the user can usually interrupt with Ctrl-C, unless we are in an automated setting, of course.  I agree that we should discuss that more...

As this is a bug story, I just made sure that the existing behavior is preserved ie., a very short timeout for the auto-update check.